### ISSUE TITLE
Include meta-metadata in individual asset endpoint responses

### ISSUE THREAD

**Issue by jwodder** [dandi/dandi-archive#673]
When listing assets via the `/dandisets/{versions__dandiset__pk}/versions/{versions__version}/assets/` endpoint, the following fields are returned:

* `asset_id`
* `blob`
* `zarr`
* `path`
* `size`
* `created`
* `modified`

However, when getting information about an individual asset via the `/dandisets/{versions__dandiset__pk}/versions/{versions__version}/assets/{asset_id}/` or `/assets/{asset_id}/` endpoint, only the asset metadata is returned.  This metadata includes values for the `asset_id`, `path`, and `size` fields, but it does not include a `created`, `blob`, or `zarr` field, and neither of the metadata fields `dateModified` and `blobDateModified` corresponds to `modified`.  This makes getting all of the information about an asset more awkward than it should have to be.

I would therefore like to request that the `/dandisets/{versions__dandiset__pk}/versions/{versions__version}/assets/{asset_id}/` endpoint (and `/assets/{asset_id}/`, if possible) return the same structure as returned when listing assets but with an added "metadata" field containing the current response structure.

**Comment by dchiquito** [comment]
I believe we made this change in #236 at @satra's request, for version metadata as well as assets. There is a `/dandisets/{versions__dandiset__pk}/versions/{versions__version}/info/` endpoint that returns the detailed serialization for the Version info. We can add something similar for assets, or we can revert the change and return the serialization instead of the metadata.

**Comment by satra** [comment]
@dchiquito  is correct and we don't want to move back to that state of returning both things in the asset metadata endpoint. 
the assets endpoint returns a paginated json structure, while the assets/<uuid>/ endpoint returns jsonld. this was intentional to ensure that we did not have to force everything to be jsonld. this is the same pattern used for the dandisets and assets. 

the actual uri that's encoded in the asset metadata identifier `dandiasset:6f8a1351-5b73-4d5d-beb2-a6c8bb5fb2fe` is important as it is interpretable by a jsonld interpreter. for example here: https://tinyurl.com/y4lky9d2

i'm not sure which part of this is making it more awkward, since most of the information is contained in the metadata record. blob/zarr shouldn't matter as it should be in the encodingFormat field of the metadata.

**Comment by dchiquito** [comment]
@jwodder are the fixes for https://github.com/dandi/dandi-api/issues/676 sufficient? If not, we can always add a `/dandisets/{versions__dandiset__pk}/versions/{versions__version}/assets/{asset_id}/info/` endpoint that returns the detailed Django serialization.

**Comment by jwodder** [comment]
@dchiquito I'd prefer an `/info/` endpoint (including a non-Dandiset-aware `/assets/{asset_id}/info/` endpoint).

**Comment by yarikoptic** [comment]
@waxlamp -- great that we are currently thinking about the same!!! indeed we are bombarding API e.g.

```
2022-07-26T11:49:19-0400 [DEBUG   ] httpx._client: HTTP Request: GET https://api.dandiarchive.org/api/assets/29b2c59a-3c93-458d-a9a2-e7d75da23c20/ "HTTP/1.1 200 OK"
2022-07-26T11:49:19-0400 [DEBUG   ] backups2datalad: Dandiset 000026: sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS29_stain-NeuN_chunk-50_SPIM.ome.tif: metadata unchanged; not taking any further action
2022-07-26T11:49:19-0400 [DEBUG   ] httpx._client: HTTP Request: GET https://api.dandiarchive.org/api/assets/40de1e27-ac26-4029-bb4e-10fe8b7e8841/ "HTTP/1.1 200 OK"
2022-07-26T11:49:19-0400 [DEBUG   ] backups2datalad: Dandiset 000026: sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS29_stain-NeuN_chunk-50_SPIM.json: metadata unchanged; not taking any further action
```
and that is actually at large unrelated to this issue AFAIK (which deals with "per asset" access point). Inclusion of the metadata into listing of assets (via /assets or /paths) is a "known" super-old issue #100 which was closed/generalized into another old issue #397. Let's may be discuss there instead, and let this particular (per asset endpoint) RiP
